### DISCUSSION TITLE
V should give a deprecation message for `$x` style string interpolation while it still works.

### DISCUSSION THREAD

**Discussion by JalonSolov** [vlang/v#18617] [Ideas]
Without the message, people may not know they need to change it, or that they can just use `v fmt` to take care of it for them.

Instead, it will just stop working one day, catching them by surprise.

**Comment by sanathusk** [comment]
Also , the new {x} syntax seems to be broken as on 20-Nov-2022 on version 0.3.2 4585488. Only the old syntax of $x or ${x} works.

**Comment by JalonSolov** [comment]
`{}` was deemed to be too problematic for many reasons.  It was reverted to `${}` only a few days after it was changed.

However, V used to support both `$` alone for simple things (and still does, for now), and `${}` for more complex things, but it is only going to use `${}` moving forward.
